### DISCUSSION TITLE
Reports and Alerts - Indicates reports are sent, but no e-mail are sent.

### DISCUSSION THREAD

**Discussion by JustNuAB** [apache/superset#21762] [Q&A / Help]
**Hi Mind hive!**

For starters, I'm a not a developer, but yet a happy amateur. So please go easy with me... 

I have run in to some problems regarding sending out emails from Superset. In the frontend everything looks ok, and indicates that the reports are sent but In the backend i got this status:

```
[2022-10-10 17:23:00,397: INFO/ForkPoolWorker-1] Would send notification for alert xxxx xxxx, to {"target": "xxxx@xxxxx.xx"}
[2022-10-10 17:23:00,438: INFO/ForkPoolWorker-1] Task reports.execute[690e2f42-4299-416e-b809-6ca65f0b0506] succeeded in 0.4011790730000939s: None
```
The set up is a Docker on OSX from here: https://superset.apache.org/docs/installation/installing-superset-using-docker-compose

I really could use som guidance on how to move forward, please advice.

Kind regards
/Martin

**Comment by sfirke** [comment]
The `would send notification` seems like a hint that you need to set `ALERT_REPORTS_NOTIFICATION_DRY_RUN` to False.  From https://superset.apache.org/docs/installation/alerts-reports/.

Perhaps you set it to False in config.py and you have it as True in superset_config.py or superset_config_docker.py, both of which would override config.py.  Here is the code that generates that message: https://github.com/apache/superset/blob/master/superset/reports/commands/execute.py#L405-L410 and it depends on that flag. Source: https://github.com/apache/superset/pull/14267

**Comment by JustNuAB** [comment]
Hello! It is already set to false. Is there a way to validate what configuration that are set, via terminal/bash?

Vänligen
Martin Ghaoui

11 okt. 2022 kl. 19:32 skrev Sam Firke ***@***.***>:

﻿

The would send notification seems like a hint that you need to set ALERT_REPORTS_NOTIFICATION_DRY_RUN to False. From https://superset.apache.org/docs/installation/alerts-reports/.

**Reply by sfirke** [reply]
I updated my answer to address this.

**Reply by JustNuAB** [reply]
Ok, I found the problem, the superset_config.py value EMAIL_NOTIFICATIONS = True did not overwrite EMAIL_NOTIFICATIONS = False in the config.py for some reason. All other values were overwritten.
